docs/batch-changes/bulk-operations-on-changesets.mdx

```diff
@@ -36,6 +36,7 @@ Below is a list of supported bulk operations for changesets and the conditions w
 | **Merge (experimental)** | Merge the selected changesets on code hosts. Some changesets may be unmergeable due to their states, which does not impact the overall bulk operation. Failed merges are listed under the bulk operations tab. In the confirmation modal, you can opt for a squash merge strategy, available on GitHub, GitLab, and Bitbucket Cloud. For Bitbucket Server/Data Center, only regular merges are performed |
 | **Close**                | Close the selected changesets on the code hosts                                                                                                                                                                                                                                                                                                                                                          |
 | **Publish**              | Publishes the selected changesets, provided they don't have a [`published` field](/batch-changes/batch-spec-yaml-reference#changesettemplatepublished) in the batch spec. You can choose between draft and normal changesets in the confirmation modal                                                                                                                                       |
+| **Push-only**           | Pushes code changes to a new branch on a code host without making a merge request. Available on GitHub and GitLab only. |
```

Comment:
Note: the bulk operation is called "push" only (verb is active), whereas the non-bulk-op (not documented in this PR) and the reference to the state of the changeset is "pushED" only (e.g. "Look at this pushed-only changeset I made").